### ISSUE TITLE
`invalidate` won't invalidate unless another chain tip has been found with the same height

### ISSUE THREAD

**Issue by fuxingloh** [DeFiCh/whale#207]
#### What would you like to be added:

By design, this prevents accidental invalidation of a large chain fork. But this proves to be problematic as a chain might be invalidated and a new chain height might not be found yet.

/area module-indexer

**Comment by fuxingloh** [comment]
/triage accepted
/assign @fuxingloh 

To allow `<10` blocks reorg

**Comment by monstrobishi** [comment]
need more clarity on what is needed, to implement `To allow <10 blocks reorg` ?

**Comment by fuxingloh** [comment]
> need more clarity on what is needed, to implement `To allow <10 blocks reorg` ?

Yup, to implement the ability to invalidate if the chain tip is at a different height. `<10` height differences for example, could be lower.
